package.json

```diff
@@ -51,6 +51,7 @@
   "dependencies": {
     "grommet-icons": "^4.5.0",
     "hoist-non-react-statics": "^3.2.0",
+    "locale-currency": "^0.0.2",
```

Comment:
![image](https://user-images.githubusercontent.com/6320236/117204267-51e76a00-adad-11eb-9ae6-3cb2bd08f82c.png)

Doesn't look too bad, yet another package to maintain.

Reply:
this package is a simple map of locale to currency, if you are concerned about the extra dependency I think there are maps available out there

```
{
   "pt-BR": "BRL",
  "pt": "EUR",
  "en-UK": "EUR",
  "en-US": "USD",
  "en": "USD"
}
```